articles/active-directory/manage-apps/application-proxy-configure-native-client-application.md

```diff
@@ -54,7 +54,7 @@ You now need to register your application in Azure AD, as follows:
    - To target only business or educational customers, select **Accounts in any organizational directory**.
    - To target the widest set of Microsoft identities, select **Accounts in any organizational directory and personal Microsoft accounts**.
 
-1. In the **Redirect URI** heading, select **Public client (mobile & desktop)**, and then type the redirect URI for your application.
+1. In the **Redirect URI** heading, select **Public client (mobile & desktop)**, and then type the redirect URI **https://login.microsoftonline.com/common/oauth2/nativeclient** for your application.
```

Comment:
>In the **Redirect URI** heading, select **Public client (mobile & desktop)**, and then type the redirect URI **https://login.microsoftonline.com/common/oauth2/nativeclient** for your application. [](start = 3, length = 195)

Steps have changed
1. Click on the **Authentication Blade**
1. Select "Mobile and desktop applications"
1. From the list of Redirect URIs presented, select **[https://login.microsoftonline.com/common/oauth2/nativeclient](https://login.microsoftonline.com/common/oauth2/nativeclient)**
1. Press the **Configure** button to save the redirect Uri.

Reply:
It's fine for me.